### ISSUE TITLE
Listing crashed trials

### ISSUE THREAD

**Issue by off99555** [determined-ai/determined#2673]
Is there a way to list trials that crashed?
Use case is to find bug when some combinations of parameters do not work together.
Example use case: I search optimizers between rmsprop, adadelta, adagrad, adam, nadam, sgd.
I found that the top 100 visualizations do not have nadam as optimizer at all. The first thought is that maybe nadam is bad optimizer. But actually it's because nadam cannot be used with learning rate schedule in keras. So it crashes every time I use learning rate schedule. When debugging the experiment locally I didn't see this because I didn't try learning rate scheduler with every optimizer.

Actually if the button in this UI is clickable, it would be an easy fix (so that I can sort to see trial that has errors).
![image](https://user-images.githubusercontent.com/15215732/124694594-ceacd680-df0b-11eb-8081-613c8c2f3804.png)

**Comment by vishnu2kmohan** [comment]
Thanks for the suggestion @off99555, we'll look into adding the ability to filter/sort by state.
